src/main/resources/org/damap/base/db/changeLog-4.x/changeLog-4.x.yaml

```diff
@@ -0,0 +1,7 @@
+databaseChangeLog:
+  - include:
+      file: org/damap/base/db/changeLog-4.x/changeLog-4.0.0_1.yaml
+  - include:
+      file: org/damap/base/db/changeLog-4.x/changeLog-4.0.0_2.yaml
+  - include:
+      file: org/damap/base/db/changeLog-4.x/changeLog-4.0.0_3.yaml
```

Comment:
Small mistake, this should be 4.0.3_1.yaml.

Reply:
Updated to take a previous change log into account. Also updated the id to 13

Reply:
Perfect